mlir/include/mlir/Dialect/OpenACC/OpenACCOps.td

```diff
@@ -1101,7 +1122,8 @@ def OpenACC_SerialOp : OpenACC_Op<"serial",
 
   let assemblyFormat = [{
     oilist(
-        `dataOperands` `(` $dataClauseOperands `:` type($dataClauseOperands) `)`
+        `combined` `(` custom<CombinedConstructs>($combined) `)`
```

Comment:
I don't think you want this in the oilist. It must appear just after the op name if present.

Reply:
The oilist is ordered - so "combined" does appear right after the op name if present.

Reply:
Done